runbotics/runbotics-scheduler/src/scheduler-database/credential/criteria/credential.criteria.ts

```diff
@@ -0,0 +1,25 @@
+import { NumberFilter } from '#/utils/specification/filter/number-filter/number.filter';
+import { StringFilter } from '#/utils/specification/filter/string-filter/string.filter';
+
+import { DatetimeFilter } from '#/utils/specification/filter/datetime-filter/datetime-filter';
+import { Criteria } from '#/utils/specification/criteria/criteria';
+import { UUIDFilter } from '#/utils/specification/filter/uuid-filter/uuid.filter';
+
+export class CredentialCriteria extends Criteria {
+    id = new NumberFilter();
```

Comment:
same, there should be StringFilter() or even UUIDFilter

Reply:
yep, UUIF filter, fair point